homeassistant/components/home_connect/select.py

```diff
@@ -59,24 +73,31 @@ async def async_setup_entry(
 class HomeConnectProgramSelectEntity(HomeConnectEntity, SelectEntity):
     """Select class for Home Connect programs."""
 
+    entity_description: HomeConnectSensorEntityDescription
+
     def __init__(
         self,
         coordinator: HomeConnectCoordinator,
         appliance: HomeConnectApplianceData,
-        desc: SelectEntityDescription,
+        desc: HomeConnectSensorEntityDescription,
     ) -> None:
         """Initialize the entity."""
         super().__init__(
             coordinator,
             appliance,
             desc,
         )
+        self.start_on_select = desc.key == EventKey.BSH_COMMON_ROOT_ACTIVE_PROGRAM
         self._attr_options = [
             PROGRAMS_TRANSLATION_KEYS_MAP[program.key]
             for program in appliance.programs
             if program.key != ProgramKey.UNKNOWN
+            and (
```

Comment:
We can remove the parenthesis.

Reply:
Now that the above suggestion has been applied, as the if statement contains `and` and `or`, ruff suggest me to use the parenthesis.

Reply:
Ok.